content/a/tuba-jednosci-wrestlingowej-gallery.toml

```diff
@@ -0,0 +1,8 @@
+1 = { path = "tjw-01.webp", caption = "Signatures.", source = "M3n747"}
```

Comment:
Consider adding `skip_art` to these ([docs](https://github.com/tpwres/teczki-polskiego-wrestlingu/blob/main/doc/GALLERY.md#gallery-notation)) - up to you.